src/Engine/Hymn.cpp

```diff
@@ -222,6 +222,20 @@ void ActiveHymn::Render(Entity* camera, bool soundSources, bool particleEmitters
     }
 }
 
+Entity* ActiveHymn::GetEntityByGUID(int GUID) {
+
+    const std::vector<Entity*> entities = Hymn().world.GetEntities();
```

Comment:
Don't forget an ampersand, otherwise it will not be a reference and we make a copy!

Reply:
Like this: 
const std::vector<Entity*> &entities = Hymn().world.GetEntities(); 
?

Reply:
Yup.